packages/aws-client/src/S3.ts

```diff
@@ -152,6 +153,35 @@ export const s3ObjectExists = (params: { Bucket: string, Key: string }) =>
       throw e;
     });
 
+/**
+ * Wait for an object to exist in S3
+ *
+ * @param {Object} params
+ * @param {string} params.bucket
+ * @param {string} params.key
+ * @param {number} [params.interval=1000]
+ * @param {number} [params.timeout=30000]
+ * @returns {Promise<undefined>}
+ */
```

Comment:
Do we want to add some descriptions to the params and return?

Reply:
Done: 66cc746b9